components/user_list.tsx

```diff
@@ -49,16 +69,18 @@ export default class UserList extends React.PureComponent {
         let content;
         if (users == null) {
             return <LoadingScreen/>;
-        } else if (users.length > 0) {
-            content = users.map((user, index) => {
+        } else if (users.length > 0 && RowComponentType && this.props.actionProps) {
+            content = users.map((user: UserProfile, index: number) => {
+                const {actionUserProps, extraInfo} = this.props;
+                const userId = user.id;
                 return (
                     <RowComponentType
                         key={user.id}
                         user={user}
-                        extraInfo={this.props.extraInfo[user.id]}
+                        extraInfo={extraInfo ? extraInfo[userId] : {}}
                         actions={this.props.actions}
                         actionProps={this.props.actionProps}
-                        actionUserProps={this.props.actionUserProps[user.id]}
+                        actionUserProps={actionUserProps ? actionUserProps[userId] : {}}
                         index={index}
                         totalUsers={users.length}
                         userCount={(index >= 0 && index < Constants.TEST_ID_COUNT) ? index : -1}
```

Comment:
lets remove the `Constants.TEST_ID_COUNT` check altogether since until now it was defined anyways its safe to assume we didn't need that. instead of defining our self one.